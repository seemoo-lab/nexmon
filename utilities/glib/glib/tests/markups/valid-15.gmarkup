<test>&lt;bla>bla</test>
