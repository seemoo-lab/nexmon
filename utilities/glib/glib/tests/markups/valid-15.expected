ELEMENT 'test'
  TEXT '<bla>bla'
END 'test'
